x-pack/plugins/spaces/server/spaces_client/spaces_client.mock.ts

```diff
@@ -28,6 +28,7 @@ const createSpacesClientMock = () =>
     }),
     create: jest.fn().mockImplementation((space: Space) => Promise.resolve(space)),
     update: jest.fn().mockImplementation((space: Space) => Promise.resolve(space)),
+    createSavedObjectFinder: jest.fn(), // TODO: import from point in time finder mock from core
```

Comment:
When I try to import `src/core/server/saved_objects/service/lib/point_in_time_finder.mock.ts` here I am getting listing errors. Any idea how to use the mock?

Reply:
I would use the function provided by the SOR mock. This seems to work without causing linting errors:
```diff
diff --git a/x-pack/plugins/spaces/server/spaces_client/spaces_client.mock.ts b/x-pack/plugins/spaces/server/spaces_client/spaces_client.mock.ts
index 05a23d54cfb..44b3fbbcb5c 100644
--- a/x-pack/plugins/spaces/server/spaces_client/spaces_client.mock.ts
+++ b/x-pack/plugins/spaces/server/spaces_client/spaces_client.mock.ts
@@ -5,12 +5,15 @@
  * 2.0.
  */
 
+import { savedObjectsRepositoryMock } from 'src/core/server/mocks';
+
 import type { Space } from '../../common';
 import { DEFAULT_SPACE_ID } from '../../common/constants';
 import type { SpacesClient } from './spaces_client';
 
-const createSpacesClientMock = () =>
-  ({
+const createSpacesClientMock = () => {
+  const repositoryMock = savedObjectsRepositoryMock.create();
+  return {
     getAll: jest.fn().mockResolvedValue([
       {
         id: DEFAULT_SPACE_ID,
@@ -28,10 +31,11 @@ const createSpacesClientMock = () =>
     }),
     create: jest.fn().mockImplementation((space: Space) => Promise.resolve(space)),
     update: jest.fn().mockImplementation((space: Space) => Promise.resolve(space)),
-    createSavedObjectFinder: jest.fn(), // TODO: import from point in time finder mock from core
+    createSavedObjectFinder: repositoryMock.createPointInTimeFinder,
     delete: jest.fn(),
     disableLegacyUrlAliases: jest.fn(),
-  } as unknown as jest.Mocked<SpacesClient>);
+  } as unknown as jest.Mocked<SpacesClient>;
+};
 
 export const spacesClientMock = {
   create: createSpacesClientMock,
```

Reply:
Amazing, thanks!